hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -4704,6 +4707,14 @@ private String reconfOzoneReadOnlyAdmins(String newVal) {
     return String.valueOf(newVal);
   }
 
+  private String reconfOzoneKeyDeletingLimitPerTask(String newVal) {
+    getConfiguration().set(OZONE_KEY_DELETING_LIMIT_PER_TASK, newVal);
```

Comment:
Please add the check that newValue should be a positive value.

Reply:
Done.

Reply:
I think may we can provide a `conf.getUnsignedInt()` for those configurations which must be larger than 0. we can do some check in `conf.getUnsignedInt()`

Reply:
You mean conf.setUnsignedInt?   It's a good idea. But the configuration class is in hadoop common module.